keyboards/primekb/meridian/meridian.c

```diff
@@ -35,19 +35,19 @@ void keyboard_post_init_user(void) {
 bool led_update_kb(led_t led_state) {
     bool res = led_update_user(led_state);
     if (res) {
- //       writePin(B12, !led_state.caps_lock);  //Un-comment this line to enable in-switch capslock indicator
+        writePin(B12, !led_state.caps_lock);
     if (led_state.caps_lock) {
-        rgblight_setrgb_at(0, 255, 0, 0); //green
+        rgblight_setrgb_at(255, 255, 255, 0);
```

Comment:
I think that the rgblight layers feature will handle this better, as it runs the code from `rgblight_set` itself, so it is called after animations are written to the array, but right before they're sent to the LEDs.